Adafruit_SHT31.cpp

```diff
@@ -29,24 +29,30 @@
 
 #include "Adafruit_SHT31.h"
 
+#define SHT31_READSTATUS 0xF32D  /**< Read Out of Status Register */
+#define SHT31_CLEARSTATUS 0x3041 /**< Clear Status */
+#define SHT31_SOFTRESET 0x30A2   /**< Soft Reset */
+#define SHT31_HEATEREN 0x306D    /**< Heater Enable */
```

Comment:
please put these back in the header

Reply:
these are really private commands, I think they shouldn't be polluting the headers. they allow e.g. bypassing the API of the class. anyway, reverted as requested.

Reply:
thank u!